[2022-07-08 06:04:46.973] [remoteagent] [info] Extension host agent started.
[2022-07-08 06:04:47.138] [remoteagent] [info] [127.0.0.1][a1d1480e][ManagementConnection] New connection established.
[2022-07-08 06:04:47.146] [remoteagent] [info] [127.0.0.1][1b11444e][ExtensionHostConnection] New connection established.
[2022-07-08 06:04:47.151] [remoteagent] [info] [127.0.0.1][1b11444e][ExtensionHostConnection] <12795> Launched Extension Host Process.
[2022-07-08 06:04:55.096] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateIcon (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-08 06:04:55.098] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateTitle (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-08 06:05:27.964] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: terminated by itself with code null, signal: SIGSEGV
[2022-07-08 06:05:51.030] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateTitle (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-08 06:24:10.373] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /root)
[2022-07-08 06:24:10.374] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[2022-07-08 07:30:02.097] [remoteagent] [info] Getting Manifest... formulahendry.vscode-mysql
[2022-07-08 07:30:02.477] [remoteagent] [info] Installing extension: formulahendry.vscode-mysql
[2022-07-08 07:30:03.620] [remoteagent] [info] Downloaded extension: formulahendry.vscode-mysql /root/.vscode-server/data/CachedExtensionVSIXs/formulahendry.vscode-mysql-0.4.1
[2022-07-08 07:30:04.137] [remoteagent] [info] Extracted extension to /root/.vscode-server/extensions/.b7e7f3fc-e9df-4899-9b87-bb5afbc9db67: formulahendry.vscode-mysql
[2022-07-08 07:30:04.146] [remoteagent] [info] Renamed to /root/.vscode-server/extensions/formulahendry.vscode-mysql-0.4.1
[2022-07-08 07:30:04.148] [remoteagent] [info] Extracting completed. formulahendry.vscode-mysql
[2022-07-08 07:30:04.149] [remoteagent] [info] Extension installed successfully: formulahendry.vscode-mysql
[2022-07-08 07:42:43.851] [remoteagent] [info] [127.0.0.1][a1d1480e][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-07-08 07:42:43.890] [remoteagent] [info] [127.0.0.1][1b11444e][ExtensionHostConnection] <12795> Extension Host Process exited with code: 0, signal: null.
[2022-07-08 07:42:43.890] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-08 07:43:13.059] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-08 07:43:13.059] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-08 07:43:13.214] [remoteagent] [info] [127.0.0.1][7ba9be8e][ManagementConnection] New connection established.
[2022-07-08 07:43:13.276] [remoteagent] [info] [127.0.0.1][e39fed8c][ExtensionHostConnection] New connection established.
[2022-07-08 07:43:13.290] [remoteagent] [info] [127.0.0.1][e39fed8c][ExtensionHostConnection] <17337> Launched Extension Host Process.
[2022-07-08 07:43:16.177] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateTitle (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-08 07:45:39.188] [remoteagent] [info] [127.0.0.1][7ba9be8e][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-07-08 07:45:39.198] [remoteagent] [info] [127.0.0.1][e39fed8c][ExtensionHostConnection] <17337> Extension Host Process exited with code: 0, signal: null.
[2022-07-08 07:45:39.199] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-08 07:45:39.199] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-08 07:45:48.330] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-08 07:45:48.330] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-08 07:45:48.517] [remoteagent] [info] [127.0.0.1][6d23a220][ManagementConnection] New connection established.
[2022-07-08 07:45:48.536] [remoteagent] [info] [127.0.0.1][b88c9b5a][ExtensionHostConnection] New connection established.
[2022-07-08 07:45:48.545] [remoteagent] [info] [127.0.0.1][b88c9b5a][ExtensionHostConnection] <17503> Launched Extension Host Process.
[2022-07-08 07:45:55.124] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateIcon (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-08 07:46:30.867] [remoteagent] [info] [127.0.0.1][6d23a220][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-07-08 07:46:31.013] [remoteagent] [info] [127.0.0.1][b88c9b5a][ExtensionHostConnection] <17503> Extension Host Process exited with code: 0, signal: null.
[2022-07-08 07:46:31.013] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-08 07:46:31.013] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-08 07:48:07.506] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-08 07:48:07.506] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-08 07:48:07.638] [remoteagent] [info] [127.0.0.1][9b721974][ManagementConnection] New connection established.
[2022-07-08 07:48:07.665] [remoteagent] [info] [127.0.0.1][b00ab1a5][ExtensionHostConnection] New connection established.
[2022-07-08 07:48:07.672] [remoteagent] [info] [127.0.0.1][b00ab1a5][ExtensionHostConnection] <17785> Launched Extension Host Process.
[2022-07-08 07:48:23.198] [remoteagent] [info] [127.0.0.1][9b721974][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-07-08 07:48:23.224] [remoteagent] [info] [127.0.0.1][b00ab1a5][ExtensionHostConnection] <17785> Extension Host Process exited with code: 0, signal: null.
[2022-07-08 07:48:23.224] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-08 07:48:23.224] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-08 07:48:32.359] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-08 07:48:32.359] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-08 07:48:32.563] [remoteagent] [info] [127.0.0.1][988e03a6][ManagementConnection] New connection established.
[2022-07-08 07:48:32.611] [remoteagent] [info] [127.0.0.1][26bbc9b5][ExtensionHostConnection] New connection established.
[2022-07-08 07:48:32.624] [remoteagent] [info] [127.0.0.1][26bbc9b5][ExtensionHostConnection] <17948> Launched Extension Host Process.
[2022-07-08 07:53:32.359] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-08 09:42:35.952] [remoteagent] [info] [127.0.0.1][988e03a6][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-08 09:42:35.955] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/root/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/server/node/server.main.js:143:7914)
    at process.emit (node:events:390:28)
[2022-07-08 11:42:43.309] [remoteagent] [info] [127.0.0.1][988e03a6][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-08 11:42:43.310] [remoteagent] [info] [127.0.0.1][0dc476a1][ManagementConnection] New connection established.
[2022-07-08 11:42:43.349] [remoteagent] [info] [127.0.0.1][4ae23f1e][ExtensionHostConnection] New connection established.
[2022-07-08 11:42:43.359] [remoteagent] [info] [127.0.0.1][4ae23f1e][ExtensionHostConnection] <22055> Launched Extension Host Process.
[2022-07-08 11:47:43.387] [remoteagent] [info] [127.0.0.1][988e03a6][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-08 11:47:43.603] [remoteagent] [info] [127.0.0.1][26bbc9b5][ExtensionHostConnection] <17948> Extension Host Process exited with code: 0, signal: null.
[2022-07-08 12:11:25.753] [remoteagent] [info] [127.0.0.1][0dc476a1][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-07-08 12:11:25.844] [remoteagent] [info] [127.0.0.1][4ae23f1e][ExtensionHostConnection] <22055> Extension Host Process exited with code: 0, signal: null.
[2022-07-08 12:11:25.844] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-08 12:16:25.844] [remoteagent] [info] Last EH closed, shutting down
